tests/test_topy.py

```diff
@@ -0,0 +1,136 @@
+import os
+import tempfile
+
+import ansys.fluent.core as pyfluent
+
+
+def test_single_jou(with_launching_container):
+
```

Comment:
@hpohekar are we asserting that the Python file has been created?

Reply:
@seanpearsonuk Yes, we are reading generated file contents by `returned = f.readlines()` and by using `assert returned` we are confirming that `returned` is non empty.

Reply:
@seanpearsonuk Yes, perfect catch! Now we are asserting that Python file has been generated for 1st and 2nd test.

We can not assert this for multiple journals because then it gives `invalid_argument error` for `os` as it can not read the generated file path due to concatenation of 2 file paths together.

Reply:
We are asserting that the Python file has been created for single journal name. Now we are resolving issues related to journal names as list of strings. We will update this code accordingly soon.

Reply:
We have updated the code now.

Reply:
Thanks @hpohekar I was looking at a version that didn't have the Python file checks - this looks good now :)